contracts/CO30_Rob_BomuGaru.swu_public_salman_map/edit_me/briefing/mission.html

```diff
@@ -0,0 +1,4 @@
+Synixe has been hired by the Federal Iraqi Government to support their efforts to re-take the city of Al Salman from forces belonging to the resurgent Islamic State (ISIS).
+This will be the third battle for the city in under 6 months. With both forces spent, Synixe has been brought in to supplement the Iraqi National Army. Additionally, Synixe has been hired
+for it's expertise in EOD. The Iraqi army believes that Al Salman can be re-taken however, intel reports suggest ISIS has mined key routes within and around the city. Synixe is to 
+dispose of this ordance while keeping damage to the cities infrastructure to a minimum. The disposal of all mines within the AO will allow for the Iraqi Army to bring in heavy equipment, and open up key logistical routes to other AO's 
```

Comment:
```suggestion
This will be the third battle for the city in under 6 months. With both forces spent, Synixe has been brought in to supplement the Iraqi National Army. Additionally, Synixe has been hired for it's expertise in EOD. The Iraqi army believes that Al Salman can be re-taken however, intel reports suggest ISIS has mined key routes within and around the city. Synixe is to dispose of this ordnance while keeping damage to the cities infrastructure to a minimum. The disposal of all mines within the AO will allow for the Iraqi Army to bring in heavy equipment, and open up key logistical routes to other AO's 
```
Needs to be a single line or it'll show up weird in Arma